src/api/operator/types.ts

```diff
@@ -19,8 +19,14 @@ export interface FeeInformation {
 
 export type OrderKind = 'sell' | 'buy'
 
-export type OrderStatus = 'open' | 'filled' | 'cancelled' | 'expired' | 'signing'
-export type RawOrderStatusFromAPI = 'presignaturePending' | 'open' | 'fullfilled' | 'cancelled' | 'expired'
+export type OrderStatus = 'open' | 'filled' | 'cancelled' | 'cancelling' | 'expired' | 'signing'
+export type RawOrderStatusFromAPI =
+  | 'presignaturePending'
+  | 'open'
+  | 'fullfilled'
+  | 'cancelled'
+  | 'cancelling'
```

Comment:
As this 'cancelling' doesn't come from the API yet, maybe we can omit it or add a comment about it.

Reply:
Great catch!